scilpy/tracking/tracker.py

```diff
@@ -0,0 +1,173 @@
+# -*- coding: utf-8 -*-
```

Comment:
Could you use the same structure of documentation as in local_tracking

Reply:
Done in d69bef2